### PR TITLE
chore: librarian time check

### PR DESCRIPTION
**Pull request by manisha1997**
Removing run_openapi_generator from for loop which was causing too much delay with pipelines

### CODE REVIEW

File: scripts/build_twilio_library.py [hunk 1]
```diff
@@ -50,6 +50,18 @@ def generate(spec_folder: str, spec_files: List[str], output_path: str, language
...
+        run_openapi_generator(parent_dir, language)
+        print(f'Code generation completed at {output_path}')
+    elif spec_files[0] not in generateForLanguages:
+        print(f'Generating {output_path} from {spec_folder}')
+        run_openapi_generator(parent_dir, language)
+        print(f'Code generation completed at {output_path}')
+    if language in CLEANUP_IMPORT_LANGUAGES:
+        remove_unused_imports(output_path, language)
+    if language in REMOVE_DUPLICATE_IMPORT_LANGUAGES:
+        remove_duplicate_imports(output_path, language)
```
Comment (sbansla) [inline scripts/build_twilio_library.py:65]: try to create a different PR which has less commits. This PR has 9 commits.
Changes looks good, I have tested code is getting generated within 20 seconds.

File: scripts/build_twilio_library.py [hunk 2]
```diff
@@ -62,18 +74,17 @@ def generate_domain_for_language(spec_file: str, config_path: str, spec_folder:
             'arrayItemSuffix': ''
         },
     }
-
+    # print(config)
     with open(full_config_path, 'w') as f:
```
Comment (sbansla) [inline scripts/build_twilio_library.py:78]: Do we need this ?

File: scripts/build_twilio_library.py (hunk 1 above)
Comment (sbansla) [reply scripts/build_twilio_library.py:65]: It takes 20 seconds because we are passing directory, If file is passed it takes more time.

Comment (sbansla) [reply scripts/build_twilio_library.py:65]: Get approval from @AsabuHere as well.

**Review by AsabuHere — APPROVED** [review]
LGTM!

**Pull request merged**